TrafficCapture/trafficReplayer/src/main/java/org/opensearch/migrations/replay/datatypes/RawPackets.java

```diff
@@ -21,4 +20,13 @@ public boolean equals(Object o) {
         }
         return true;
     }
+
+    @Override
+    public int hashCode() {
+        int result = 29;
```

Comment:
I don't love these magic numbers, would prefer some constants here

Reply:
Discussed offline, i'll leave it up to Greg on whether or not to use `Arrays.hashCode(toArray());`

Reply:
I'm much more comfortable with 2 primes for a hash function than doing an copy.  
Honestly, I'd rather see this as a one-liner map-reduce operation like this, but most hashCode operations, especially those generated by IDEs and code generators (we have a lot of protobuf generated ones in out captureProtobufs package).
```return stream().mapToInt(Arrays::hashCode).reduce(29, (a,b) -> 31*a+b);```